.github/workflows/github_action_test_suite.yml

```diff
@@ -66,27 +68,19 @@ jobs:
       - name: Run test suite
         if: ${{ ! matrix.coverage-test }}
         working-directory: testing_mygame
+        env:
+          UNIT_TEST_SETTINGS: "--settings=settings --keepdb --parallel 4 --timing evennia"
```

Comment:
The `evennia` part here is not really part of the settings/options of the test run. It would make more sense to just have the options here, so you can do `evennia test ${{ env.UNIT_TEST_SETTINGS}} evennia` later.